backend/app/api/v1/routes/index.ts

```diff
@@ -27,6 +27,7 @@ router.get("/api/users/:user_id/help-requests", helpRequestsController.getByUser
 
 // * Help Offers
 router.post("/api/users/:user_id/help-offers", helpOffersController.create);
+router.get("/api/help-offers/:help_request_id", helpOffersController.getByRequestId)
```

Comment:
We should follow the pattern as above for ex: `/api/help-requests/:help_request_id/help-offers`